src/nss-myhostname/nss-myhostname.c

```diff
@@ -180,8 +180,6 @@ enum nss_status _nss_myhostname_gethostbyname4_r(
         /* Explicitly reset all error variables */
         *errnop = 0;
         *h_errnop = NETDB_SUCCESS;
-        h_errno = 0;
-
```

Comment:
Well, there's a certain conflict here between theory and reality...

See e70df46b9721a3d025e7a0b4ffb5893cbde5e55d